Libraries/WebSocket/__tests__/Websocket-test.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+jest
+  .autoMockOff()
```

Comment:
why do you have to turn off auto-mocking? Can't you just remove this line and continue mocking `NativeModules` just like you do below?